tests/test-cases/include-component/component-local/.gitlab-ci.yml

```diff
@@ -0,0 +1,8 @@
+---
+include:
+  - component: $CI_SERVER_HOST/$CI_PROJECT_PATH/my-component@$CI_COMMIT_SHA
```

Comment:
Seemed liked we're missing the predefined var `CI_SERVER_FQDN`
![image](https://github.com/firecow/gitlab-ci-local/assets/28011783/e3d366c1-c8a0-4534-be7b-065867fb7b57)